.gitmodules

```diff
@@ -1,18 +1,18 @@
 [submodule "tempo-operator"]
 	path = tempo-operator
-	url = https://github.com/os-observability/tempo-operator.git
+	url = https://github.com/grafana/tempo-operator.git
```

Comment:
https://github.com/grafana/tempo-operator/pull/1124 got merged after the latest upstream release, so I think either we need another upstream release or update x/net in the fork

Reply:
Can't we pin just that commit?